### DISCUSSION TITLE
What’s difference between Turborepo and Yarn v3?

### DISCUSSION THREAD

**Discussion by nhattran998** [vercel/turborepo#181] [Help] [answered]
Note I'm a newbie with monorepo - just try all of that to choose what the best for my team.

When I skim the docs of Turbo, I just saw the key feature it's the remote caching like Nx do - I guess so.
So anyone can explain to me the details of this? 
This tool will be better for what project?

Thanks for reading this thread 😄

**Comment by leerob** [comment, marked as answer]
Turborepo works _with_ Yarn monorepos, as well as NPM or PNPM. Yarn workspaces do not natively support features like local caching (or remote caching). Any time you run a build, you must recompute that work every time. With Turbo, it's able to intelligently cache build and log outputs to never recompute work that's already been completed.

**Reply by nhattran998** [reply]
Thank @leerob <3. I think It's would be nice if this push on the document for the newbie like me or everyone.

**Reply by killjoy2013** [reply]
@leerob  What are the things we can do only with a monorepo structure, we can't do only using yarn workspaces? An obvious one is remote caching. How about the others?
